File: PYTHON/PetShop.py
class PetShop:
    id = 0
    nama = ""
    harga = 0
    stok = 0
    
    def __init__(self, idProduk=0, nama="", harga=0, stok=0):
        self.id = idProduk
        self.nama = nama
        self.harga = harga
        self.stok = stok

    def setId(self, idProduk): self.id = idProduk
    def getId(self): return self.id

    def setNama(self, nama): self.nama = nama
    def getNama(self): return self.nama

    def setHarga(self, harga): self.harga = harga      
    def getHarga(self): return self.harga

    def setStok(self, stok): self.stok = stok  
    def getStok(self): return self.stok  

File: CPP/PetShop.cpp
#include <iostream>
#include <string>

using namespace std;

class PetShop
{
private:
  int id, harga, stok;
  string nama;

public:
  PetShop()
  {
    this->id = 0;
    this->nama = "";
    this->harga = 0;
    this->stok = 0;
  }

  PetShop(int id, string nama, int harga, int stok)
  {
    this->id = id;
    this->nama = nama;
    this->harga = harga;
    this->stok = stok;
  }

  void setId(int id)
  {
    this->id = id;
  }

  void setNama(string nama)
  {
    this->nama = nama;
  }

  void setHarga(int harga)
  {
    this->harga = harga;
  }

  void setStok(int stok)
  {
    this->stok = stok;
  }

  int getId()
  {
    return this->id;
  }

  string getNama()
  {
    return this->nama;
  }

  int getHarga()
  {
    return this->harga;
  }

  int getStok()
  {
    return this->stok;
  }
};